Minor enhancements in Matomo doc (#752)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/matomo.mdx

| `properties.category` | `categoryName` |Required |
| `properties.price` (Multiplied by quantity) | `price` |Required |

<div class="infoBlock">

> The <code class="inline-code">Product Viewed</code> event can be observed in the destination dashboard only when followed by a <code class="inline-code">page</code> call.

Reviewer: ```suggestion
You can see the <code class="inline-code">Product Viewed</code> event in the Matomo dashboard only when it is followed by a <code class="inline-code">page</code> call.
```

Reviewer: Would something like:

"Send the `Product Viewed` event followed by a `page` event to view it in the Matomo dashboard."

be an alternative to the above statement?

Author: yup, can be. But I like the above better

| `properties.price` (Multiplied by quantity) | `price` |Optional |
| `properties.quantity` | `quantity` |Optional |

<div class="infoBlock">

> The <code class="inline-code">Product Added</code> event can be observed in the destination dashboard only when followed by an <code class="inline-code">Update Cart</code> or <code class="inline-code">Order Completed</code> event.

Reviewer: or the alternative similar to the I shared above.

|:---------------------|:-------------|:--------------|
| `properties.sku`/`properties.product_id` | `productSKU` | Required |

<div class="infoBlock">
The <code class="inline-code">Product Removed</code> event can be observed in the destination dashboard only when followed by an <code class="inline-code">Update Cart</code> or <code class="inline-code">Order Completed</code> event.
</div>

> 

Reviewer: This can be tweaked as per the above suggestions too.
